### PR TITLE
Csv upload timeout

### PR DESCRIPTION
**Pull request by mariarivera30**

### CODE REVIEW

File: pybossa/importers/importer.py [hunk 1]
```diff
@@ -177,12 +178,11 @@ def create_tasks(self, task_repo, project, **form_data):
             if found is None:
                 if validator.validate(task):
                     try:
-                        task_repo.save(task)
                         n += 1
+                        task_repo.save(task, clean_project=(n == n_tasks))
```
Comment (fiorda) [inline pybossa/importers/importer.py]: if the csv import times out, or one of the tasks fails to save, the cache is not going to be cleared

Comment (mariarivera30) [reply pybossa/importers/importer.py]: ok so I can call the clean from here then in the exception block.

Comment (mariarivera30) [reply pybossa/importers/importer.py]: done

File: pybossa/importers/importer.py [hunk 2]
```diff
@@ -177,12 +180,12 @@ def create_tasks(self, task_repo, project, **form_data):
...
                     except Exception as e:
+                        cached_projects.clean_project(project.id)
```
Comment (fiorda) [inline pybossa/importers/importer.py]: can we instead do this outside of the `for` loop?

**Pull request merged**